crates/compiler/gen_dev/src/generic64/x86_64.rs

```diff
@@ -1782,9 +1782,9 @@ impl Assembler<X86_64GeneralReg, X86_64FloatReg> for X86_64Assembler {
     }
 }
 
-fn shift_reg64_reg64_reg64<'a, 'r, ASM, CC>(
+fn shift_reg64_reg64_reg64<'a, ASM, CC>(
     buf: &mut Vec<'a, u8>,
-    storage_manager: &mut StorageManager<'a, 'r, X86_64GeneralReg, X86_64FloatReg, ASM, CC>,
+    storage_manager: &mut StorageManager<'a, X86_64GeneralReg, X86_64FloatReg, ASM, CC>,
```

Comment:
I think this is wrong? that lifetime did not disappear from the definition of StorageManager

Reply:
Good catch! I've replaced it with `'_`